airrohr-firmware/airrohr-firmware.ino

```diff
@@ -4614,17 +4621,16 @@ static unsigned long sendDataToOptionalApis(const String &data) {
 		send_csv(data);
 	}
 
-	if (cfg::send2sd)
-	{
+	return sum_send_time;
+}
+
+void openLoggingFile(){
```

Comment:
@MCLEANS that we needed an entirely different function might be a need for a different approach as we want the code base to remain as uniform as possible.